config/bmcweb.service.in

```diff
@@ -1,5 +1,7 @@
 [Unit]
 Description=Start bmcweb server
+StartLimitIntervalSec=30
```

Comment:
Hmm, this doesn't seem to match with what's in the commit msg:
"Allow bmcweb to be restarted 5 times over a 15 second period before stopping."

Reply:
Yep, that was a mistake in the original Comment message / PR Message. I fixed that. It now reads 
"Allow bmcweb to be restarted 4 times over a 30 second period before stopping."